src/main/java/com/sekwah/advancedportals/bukkit/PluginMessages.java

```diff
@@ -3,22 +3,35 @@
 import com.sekwah.advancedportals.bukkit.config.ConfigAccessor;
 import org.bukkit.ChatColor;
 import org.bukkit.command.CommandSender;
+import org.checkerframework.checker.regex.qual.Regex;
 
 public class PluginMessages {
     private static String WARP_MESSAGE;
+    private static String COOLDOWN_PROTECTION_MESSAGE;
+    private static String PORTAL_COOLDOWN_MESSAGE;
+    private static String NO_PERMISSION_PORTAL;
+    private static String NO_BUILD_PERMISSION;
     public boolean useCustomPrefix = false;
-    public static String customPrefix = "\u00A7a[\u00A7eAdvancedPortals\u00A7a]";
```

Comment:
Please keep the characters as \u00A7a rather than § as in the past certain systems have had problems compiling it for some reason. There was some odd legacy reason I kept it this way.

Reply:
Actually ill keep it for now, see if the issue pops back up